pkg/webhooks/middleware.go

```diff
@@ -0,0 +1,15 @@
+package webhooks
+
+import (
+	"net/http"
+	"time"
+)
+
+func timeoutHandler(h http.Handler) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		var timeoutHandler http.Handler
+		msg := "Your request has timed out."
+		timeoutHandler = http.TimeoutHandler(h, 5*time.Second, msg)
```

Comment:
There is a command line arguement `webhookTimeout`. The value of the argument should be used as the timeout, if no argument is given use default value.

Reply:
can you clear that point ?